### PR TITLE
Add only-hide

### PR DESCRIPTION
**Pull request by jamesnw**
## Description
I'm unsure of the right interface, but I realized we may want the inverse as well, and it was very simple to extend the OnlyShow element. I considered adding an `inverse` attribute, but it seemed odd. I'm definitely fine if this should be done differently (or not at all).

## Steps to test/reproduce
Check out the "Hide instead of show" portion of the demo.

### CODE REVIEW

File: only-show.js [hunk 1]
```diff
@@ -96,3 +96,16 @@ class OnlyShow extends HTMLElement {
...
+    if ("customElements" in window) {
+      customElements.define(tagName || "only-hide", OnlyHide);
+    }
+  }
+  showHide = () => {
+    this.toggleAttribute('hidden', this.matchConditions);
+  }
+}
+
+OnlyHide.register();
```
Comment (mirisuzanne) [inline only-show.js]:
```suggestion
OnlyHide.register();

```

**Review by mirisuzanne — APPROVED** [review]
I think at this point I lean towards `hide-when` and `show-when` with `has-*` attributes? But I don't feel strongly. Also ok with merging as-is.

(you should probably add yourself as a contributor on the package)

**Comment by jamesnw** [comment]
> I think at this point I lean towards `hide-when` and `show-when` with `has-*` attributes? 

I like this too, but then wonder if we would need to rename the repo? Not sure it's worth the trouble unless we're fairly certain?

**Comment by mirisuzanne** [comment]
@jamesnw On the flip side - if we do want to change it, now is the perfect time. That becomes a lot more of a problem after we start using it or telling people about it. 

Are there other API/DX reasons not to make the switch? Or a less invasive proposal we like as well? I think if we like the usability of better changing it, we should just go for it.

Another more generic API that I could imagine is something like

```html
<toggle-when has-hash="my-hash" then-hide>hide when hash is present</toggle-when>
<toggle-when has-hash="my-hash" then-show>show when hash is present</toggle-when>
```

Even more generic (too generic?):

```html
<toggle-when has-hash="my-hash" attr-off="any-attr">Remove any-attr when hash is present</toggle-when>
<toggle-when has-hash="my-hash" attr-on="any-attr">Apply any-attr when hash is present</toggle-when>
```

**Comment by jamesnw** [comment]
I'm not a fan of the very generic interface, so in my mind it's between-

```
<toggle-when has-hash="my-hash" then-hide>hide when hash is present</toggle-when>
or 
<hide-when has-hash="my-hash">hide when hash is present</hide-when>
```
and
```
<toggle-when has-hash="my-hash" then-show>show when hash is present</toggle-when>
<show-when has-has="my-hash">show when hash is present</show-when>
```

I like `show` and `hide` over `toggle`, which isn't as clearly visibility in my mind. It also puts the primary objective for using it right at the beginning. I also don't think there's a big difference for users in switching between them when they realize they want the inverse.

I think I've talked myself into `hide-when` and `show-when`, but can be convinced otherwise.

**Comment by jamesnw** [comment]
I started the move over, but didn't change the repo name over, or any of the mentions of the repo in the package, yet. Let me know if I should go ahead and do that.

**Comment by mirisuzanne** [comment]
I think show-when/hide-when is the best way to go for this. Let's finish it out.

**Comment by jamesnw** [comment]
@mirisuzanne Done. You may want to run `git remote set-url origin git@github.com:oddbird/show-when.git` in your local repo to update.

**Pull request merged**